neuralogic/core/constructs/function/function.py

```diff
@@ -83,6 +86,16 @@ class Combination(Function):
     CONCAT: "Combination"
     COSSIM: "Combination"
 
+    def get(self):
+        name = self.name.capitalize()
+
+        if name in ("Sum", "Max", "Min", "Avg", "Count"):
+            return jpype.JClass(f"cz.cvut.fel.ida.algebra.functions.aggregation.{name}")()
+        if name == "Elproduct":
+            name = "ElementProduct"
+
+        return jpype.JClass(f"cz.cvut.fel.ida.algebra.functions.combination.{name}")()
```

Comment:
Will this work with parametrized combinations? E.g., concat?

Reply:
It wont - combinations are now defined as unary functions (`F.concat(entity, axis)`) 💀💀